### DISCUSSION TITLE
骚里骚气得言语 - 格莱姆的日常

### DISCUSSION THREAD

**Discussion by giscus** [tffats/shafish_blog#88] [Announcements]
# 骚里骚气得言语 - 格莱姆的日常

Just write something about Work、Technology and Life.

https://shafish.cn/comments/

**Comment by shafishcn** [comment]
刚了解到cutefish os，结果疑似他们开发团队解散了

**Comment by shafishcn** [comment]
dilililidilililidada

**Comment by shafishcn** [comment]
一拳超人、高达00还是那么的经典

**Comment by shafishcn** [comment]
爱人先爱己，谋爱先谋生

**Reply by shafishcn** [reply]
谋生到几分再谋爱，爱己有几成再爱人？

**Comment by shafishcn** [comment]
生活无常，都是大常包小常

**Comment by marigold233** [comment]
你的mkdocs的中文是怎么做的

**Reply by marigold233** [reply]
中文搜索是怎么做的

**Reply by shafishcn** [reply]
https://squidfunk.github.io/mkdocs-material/blog/2022/05/05/chinese-search-support/

**Comment by flow2000** [comment]
mkdocs的mkdocs-material主题嘛

**Reply by shafishcn** [reply]
是他是他就是他

**Comment by shafishcn** [comment]
projector就是个鸡肋东西，快捷键跟浏览器有冲突还可以忍，一点点网络波动就要断开重连，什么玩意

**Comment by shafishcn** [comment]
发现个好东西，设计数据库表用的-[`PDManer`](https://www.yuque.com/pdmaner/docs/pdmaner-manual)。
- 安装好软件对着文档过一遍就能上手
- 能画关系图
- 支持导出ddl
- 支持导出自定义mybatisplus模板代码
- 支持多种数据库
- 还有很多小示例
相当不错的表设计工具！！

**Comment by jartins** [comment]
## 大佬 你这个留言板是咋配置的，问啥我配置之后报错，编译不过。

**Reply by shafishcn** [reply]
还不是大佬:sweat_smile:，可以先按着官方文档来试下：https://squidfunk.github.io/mkdocs-material/setup/adding-a-comment-system/

晚点我再修改下这篇mkdoc使用记录：https://shafish.cn/blog/mkdocs/

**Reply by shafishcn** [reply]
报啥错？

**Comment by jartins** [comment]
为啥

**Comment by jartins** [comment]
新年快乐

**Reply by shafishcn** [reply]
新年快乐，同乐同乐

**Comment by Wcowin** [comment]
网站很好看的配色，可以互加一下友链

**Comment by shafishcn** [comment]
关于typecho管理员密码重置：`update typecho_users set password = 'e10adc3949ba59abbe56e057f20f883e' where name = 'xxx';`，重置密码为123456

**Comment by Wcowin** [comment]
![](https://s2.loli.net/2024/06/27/9gw37T4vPASxiD8.png)
大佬，拜读了你的mkdocs教程，欢迎来加个[友链](https://wcowin.work/)

**Reply by shafishcn** [reply]
:joy: 我星期天（7号）会把这评论隐藏掉，因为图片有点占内容了。后续决定要加友链的话，我可能会规范下相关 pull 规则，再开放给需要的朋友添加。当然，我看到了你的网站，做得相当不错，功能也很丰富，一定要继续坚持写下去yo :star2: :star2:

**Comment by shenweiyan** [comment]
站点很棒，MkDocs Material 👍

**Comment by famasyang** [comment]
很好的博客，爱来自龙岗区。

**Comment by shenweiyan** [comment]
请教一下，博主的[这种](https://shafish.cn/life/)折叠的页面效果是怎么实现的，需要什么插件吗？谢谢。

**Reply by shenweiyan** [reply]
发现了，应该是 `pymdownx.tabbed` 😍

**Comment by shafishcn** [comment]
书再难背也就几个月，跑步再累也就五分钟，题目再难也会有解法，累了就听听歌，散散步，出去走走。所以，你不会且觉得难的东西一定不要躲，先弄明白后精湛，你就比别人优秀。----BV1zs42137RY

**Comment by shafishcn** [comment]
任何工作要么让你学习（learn），要么让你赚钱（earn）。如果既学不到新东西，又赚不到钱，你就应该走了。en，已经走两个月了。

**Comment by shafishcn** [comment]
有人说努力是因为没有退路，好像也挺有道理

**Comment by shafishcn** [comment]
最近内容没提交，我qiao

**Reply by shafishcn** [reply]
丢了

**Comment by shafishcn** [comment]
人类最顶级的保养：不是运动，也不是睡觉，养生重在养心。只吃眼前饭，只喝眼前水，只做眼前事，只看眼前人。

**Comment by shafishcn** [comment]
创造是抵御一切虚无主义的最佳解药。

**Comment by jartins** [comment]
你的网站域名是啥 好像打不开了
